lib/components/Shell/Shell.tsx

```diff
@@ -12,77 +11,30 @@ export { NavigationProperties, NavigationItemSeparator, NavigationItemContainer
 
 const css = classnames.bind(require('./Shell.module.scss'));
 
-/** Root theme that will be passed thru the control tree. */
-export interface ShellTheme {
-    base: string;
-
-    // Masthead
-    colorBgMasthead?: string;
-    colorBgMastheadHover?: string;
-    colorBgMastheadDisabled?: string;
-    colorTextMastheadRest?: string;
-    colorTextMastheadDisabled?: string;
-
-    // Primary buttons
-    colorBgBtnPrimaryRest?: string;
-    colorBgBtnPrimaryHover?: string;
-    colorBgBtnPrimaryDisabled?: string;
-    colorTextBtnPrimaryRest?: string;
-    colorTextBtnPrimaryDisabled?: string;
-
-    // Normal buttons
-    colorBgBtnStandardRest?: string;
-    colorBgBtnStandardHover?: string;
-    colorBgBtnStandardDisabled?: string;
-    colorTextBtnStandardRest?: string;
-    colorTextBtnStandardDisabled?: string;
-}
-
 export interface ShellProperties {
-    theme?: string | ShellTheme;
     masthead?: MastheadProperties;
     navigation?: NavigationProperties;
     children?: React.ReactNode;
-    onClick?: React.MouseEventHandler<HTMLDivElement>;
 }
 
-export function Shell({ theme, masthead, navigation, children, onClick }: ShellProperties) {
-    // backward compatibility handle string format theme
-    const shellTheme: ShellTheme = React.useMemo(() => (
-        typeof theme === 'object'
-            ? theme
-            : { base: theme }
-    ), [theme]);
-
-    if (shellTheme.base === undefined) {
-        shellTheme.base = 'light';
-    }
-
-    React.useEffect(() => {
-        document.documentElement.setAttribute('theme', shellTheme.base);

```

Comment:
Who was using this?

Reply:
This was the thing that set the theme in the DOM when it changed... since now the fluent controls shouldn't know about theme this will be moved to renderer (since we also have to add similar logic now to support customization and white labeling it makes sense that all lives in the same place)